uwblueprint/planet-read
# Store human readable languages

## Notion ticket link



[Store human-readable langauges in languages table](https://www.notion.so/uwblueprintexecs/Store-human-readable-langauges-in-languages-table-e50a5580eb794b649fdf92a6cf8921a9)



## Implementation description

-



## Steps to test

1. Upgrade db and insert data
2. Check that languages table was updated
3. Login as Angela Merkel
4. Check that languages show up correctly in the frontend (primarily in dropdowns)



## What should reviewers focus on?

- Does it work?
- Note: backend does not currently validate any languages that are passed in 

## Checklist
